checks/system/systems_data/extract_data.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import yaml
+import sys
+import os
+sys.path.append("../integration")
+from constants import *
+import sys
+import glob
+import json
+ 
+
+def main(yaml_config):
+
+    def read_config_yaml(file_path):
+        """Read a YAML configuration file and return its content."""
+        with open(file_path, 'r') as config_yaml:
+            config = yaml.safe_load(config_yaml)
+
+            return config
+
+    # Look recursively for yaml files in the config directory
+    for file_i in glob.glob(yaml_config+"/*", recursive=True):
+        if file_i.endswith("yml"):  
+            print(f"Extracting data from {file_i}...")
+            try:
+                config_yaml_data
+            except:
+                config_yaml_data = read_config_yaml(file_i)
+            else:
+                config_yaml_data.update(read_config_yaml(file_i))
+
+    test_info = {}
+    # Check for mount points to be checked
+    try:
+        mount_info = config_yaml_data[MOUNT_VARS]
+        for i in range(len(mount_info)):
+            del mount_info[i]['src']
+            del mount_info[i]['opts']
```

Comment:
```suggestion
        for m in mount_info:
            del m['src']
            del m['opts']
```
This would be the same, right?